FullCircle/FullCircle/Models/Action/Action.swift

```diff
@@ -13,6 +13,7 @@ struct Action {
     let name: String
     let description: String
     let organizationID: String
+    let organizationName: String
```

Comment:
Hey @davidlawrencer ! How is organizationName different from the name property?

Reply:
great question! is `name` the org name? if so i will remove! @msystang

Reply:
Yes! name is org name -- we can keep whichever property you feel is more intuitive! Right now I see organizationName has the actual org name on Firebase

Reply:
name here is Action name! Please disregard. Looks great!